examples/train.py

```diff
@@ -64,20 +64,30 @@ def main():
     
     np.random.seed(config["seed"])
 
-    train_size = int(0.8*len(dynamics_dataset))
-    test_size = len(dynamics_dataset) - train_size
+    dynamics_train_size = int(0.8*len(dynamics_dataset))
+    dynamics_test_size = len(dynamics_dataset) - dynamics_train_size
 
-    dynamics_train_dataset, dynamics_test_dataset = torch.utils.data.random_split(dynamics_dataset, [train_size, test_size])
+    labels_train_size = int(0.8 * len(labels_dataset))
+    labels_test_size = len(labels_dataset) - labels_train_size
+
+    dynamics_train_dataset, dynamics_test_dataset = torch.utils.data.random_split(dynamics_dataset, [dynamics_train_size, dynamics_test_size])
     dynamics_train_loader = DataLoader(dynamics_train_dataset, batch_size=config["batch_size"], shuffle=True)
     dynamics_test_loader = DataLoader(dynamics_test_dataset, batch_size=config["batch_size"], shuffle=True)
-    labels_loader = DataLoader(labels_dataset, batch_size=config["batch_size"], shuffle=True)
+
+    labels_train_dataset, labels_test_dataset = torch.utils.data.random_split(labels_dataset, [labels_train_size, labels_test_size])
+    labels_train_loader = DataLoader(labels_train_dataset, batch_size=config["batch_size"], shuffle=True)
```

Comment:
Neither of these is using `collate_fn` argument (which may not be necessary anymore, in which case feel free to ignore this comment)